docs_src/microservices/support/scheduler/Ch-Scheduler.md

```diff
@@ -42,38 +42,36 @@ The latest developed data model will be updated in the [Swagger API document](ht
-|Publisher|The client to publish the message. The client ID must be unique across all clients connecting to the same broker.(Only using when type is MQTT)|
-|topic|The topic to publish to.|
-|qos|Determines the quality of service level used to guarantee message delivery.(Only using when type is MQTT)|
-|keepAlive|Maximum time in seconds to allow the connection to the MQTT broker to remain idle.(Only using when type is MQTT)|
-|retained|Determines whether or not the MQTT broker retains the message last published.(Only using when type is MQTT)|
-|autoReconnect|Reconnect to the broker when the connection failure.(Only using when type is MQTT)|
-|connectTimeout|The time interval in seconds to allow a connect to complete.(Only using when type is MQTT)|
+
+=== "Intervals"
+    |Property|Description|
+    |---|---|
+    ||An object defining a specific "period" in time|
+    |Id|Uniquely identifies an interval, for example a UUID|
+    |Created|A timestamp indicating when the interval was created in the database|
+    |Modified|A timestamp indicating when the interval was last modified|
+    |Name |the name of the given interval - unique for the EdgeX instance|
+    |Start|The start time of the given interval in ISO 8601 format|
+    |End|The end time of the given interval in ISO 8601 format|
+    |Interval |How often the specific resource needs to be polled. It represents as a duration string. The format of this field is to be an unsigned integer followed by a unit which may be "ns", "us" (or "µs"), "ms", "s", "m", "h" representing nanoseconds, microseconds, milliseconds, seconds, minutes or hours. Eg, "100ms", "24h"|
+=== "IntervalActions"
+    |Property|Description|
+    |---|---|
+    ||The action triggered by the service when the associated interval occurs|
+    |Id|Uniquely identifies an interval action, for example a UUID|
+    |Created|A timestamp indicating when the interval action was created in the database|
+    |Modified|A timestamp indicating when the interval action was last modified|
+    |Name |the name of the interval action|
+    |Interval|associated interval that defines when the action occurs|
+    |AdminState|interval action state - either LOCKED or UNLOCKED|
+    |Protocol|Indicates which protocol should be used.  Only http is used today |
+    |Host|The host targeted by the action when it activates|
+    |Port|The port on the targeted host|
+    |Method |Indicates which Http verb should be used for the REST endpoint.(Only using when type is REST|
+    |Path|The HTTP path at the targeted host for fulfillment of the action.(Only using when type is REST)|
+    |Target|The service target which is to receive the REST call - example core-data|
+
+See [Interval](https://app.swaggerhub.com/apis/EdgeXFoundry1/support-scheduler/2.0.0#/Interval) and [IntervalAction](https://app.swaggerhub.com/apis/EdgeXFoundry1/support-scheduler/2.0.0#/IntervalAction) for more information.
```

Comment:
Since this PR is based on main, the version of the swagger file should be `2.1.0` instead of `2.0.0`
https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/2.1.0#/Interval

It's not available right now, but it will be available after this PR merged:
https://github.com/edgexfoundry/edgex-go/pull/3746